diff --git a/node_modules/react-native-agora-chat/native_src/java/com/easemob/ext_sdk/dispatch/ExtSdkHelper.java b/node_modules/react-native-agora-chat/native_src/java/com/easemob/ext_sdk/dispatch/ExtSdkHelper.java
index d11e104..254bf5d 100644
--- a/node_modules/react-native-agora-chat/native_src/java/com/easemob/ext_sdk/dispatch/ExtSdkHelper.java
+++ b/node_modules/react-native-agora-chat/native_src/java/com/easemob/ext_sdk/dispatch/ExtSdkHelper.java
@@ -1265,7 +1265,7 @@ class ExtSdkReactionOperationHelper {
         Map<String, Object> data = new HashMap<>();
         data.put("userId", operation.getUserId());
         data.put("reaction", operation.getReaction());
-        data.put("operate", operation.getOperation());
+        data.put("operate", operation.getOperation().ordinal());
         return data;
     }
 }
diff --git a/node_modules/react-native-agora-chat/package.json b/node_modules/react-native-agora-chat/package.json
index 8eb57d5..42e10c7 100644
--- a/node_modules/react-native-agora-chat/package.json
+++ b/node_modules/react-native-agora-chat/package.json
@@ -60,7 +60,7 @@
   "bugs": {
     "url": "https://github.com"
   },
-  "homepage": "",
+  "homepage": "https://www.npmjs.com/package/react-native-agora-chat",
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
